scenario/src/main/kotlin/com/cosmotech/scenario/service/ScenarioServiceImpl.kt

```diff
@@ -548,21 +547,17 @@ internal class ScenarioServiceImpl(
       workspaceId: String,
       scenarioId: String
   ): Scenario {
-    checkInternalResultDataServiceConfiguration()
-    return findScenarioById(
-        organizationId, workspaceId, scenarioId, withLastRun = true, withState = true)
+    return findScenarioById(organizationId, workspaceId, scenarioId, withState = true)
```

Comment:
Why did you remove `checkInternalResultDataServiceConfiguration` method here?

Reply:
And that's why we need review. That's a mistake.